### ISSUE TITLE
didn't work

### ISSUE THREAD

**Issue by unclehowell** [mkoshtenko/openwrt-builder#1]
➜  openwrt-builder git:(master) vagrant ssh -c 'cd ~/; ./scripts/build'
➜ Make "builder" directory.
mkdir: cannot create directory ‘builder’: File exists
➜ Download and unpack image builder.
--2021-02-06 18:38:50--  https://downloads.openwrt.org/snapshots/targets/bcm27xx/bcm2711/openwrt-imagebuilder-bcm27xx-bcm2711.Linux-x86_64.tar.xz
Resolving downloads.openwrt.org (downloads.openwrt.org)... failed: Temporary failure in name resolution.
wget: unable to resolve host address ‘downloads.openwrt.org’
tar: openwrt-imagebuilder-bcm27xx-bcm2711.Linux-x86_64.tar.xz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
➜ Create feeds config.
➜ Update feeds.
./scripts/build: 26: ./scripts/build: ./scripts/feeds: not found
➜ Build image.
make: *** No rule to make target 'image'.  Stop.
➜ Copy build.
cp: cannot stat '/home/vagrant/builder/bin/targets/bcm27xx/bcm2711/openwrt-bcm27xx-bcm2711-rpi-4-ext4-factory.img.gz': No such file or directory
Connection to 127.0.0.1 closed.

**Comment by mkoshtenko** [comment]
@unclehowell , thanks for reporting! I'll have a look

based on the report, I see multiple potential fixes/improvements:
  1. The script should stop after the first encountered failure;
  2. The script should delete the build folder if it already exists
  3. Check if openwrt download path has changed

**Comment by unclehowell** [comment]
I ended up fulfilling my objective with the following script - https://github.com/unclehowell/datro/tree/net-installer/hbnb/arm/ But this is only for arm (raspberry pi). 

What my script should do is autonomously build a disk image for the Pi SD Card. When the SD Card is inserted into the Pi it then autonomously builds a nginx or apache2 web server with the following splashpage https://gui.datro.xyz

The reason I was interested in your repo is because I wanted to get my project working on Linux wireless routers and not just home servers like Raspberry Pi and UpBoards. I thought openwrt on Pi would be a stepping stone and help me understand openwrt better. 

I did fly the guys from DD-WRT to my campus in the Caribbean to help me with this a few years ago, but after the first works package the guys did I had funding issues for this project and had to go more open-source until we have the capital to afford such talented developers and business objectives.
